docs/pages/api-docs/xgrid.md

```diff
@@ -0,0 +1,113 @@
+| <span class="prop-name">logLevel</span> | <span class="prop-type">string | false</span> | false | Allows to pass the logging level or false to turn off logging. |
+| <span class="prop-name">sortModel</span> | <span class="prop-type">SortModel</span> |   | Set the sort model of the grid. |
+| <span class="prop-name">onCellClick</span> | <span class="prop-type">(param: CellParams) => void</span> |   | Callback fired when a click event comes from a cell element. |
+| <span class="prop-name">onCellHover</span> | <span class="prop-type">(param: CellParams) => void</span> |   | Callback fired when a hover event comes from a cell element. |
+| <span class="prop-name">onRowClick</span> | <span class="prop-type">(param: RowParams) => void</span> |   | Callback fired when a click event comes from a row container element. |
+| <span class="prop-name">onRowHover</span> | <span class="prop-type">(param: RowParams) => void</span> |   | Callback fired when a hover event comes from a row container element. |
+| <span class="prop-name">onRowSelected</span> | <span class="prop-type">(param: RowSelectedParams) => void</span> |   | Callback fired when one row is selected. |
+| <span class="prop-name">onSelectionChange</span> | <span class="prop-type">(param: SelectionChangeParams) => void</span> |   | Callback fired when the selection state of one or multiple rows changes. |
+| <span class="prop-name">onColumnHeaderClick</span> | <span class="prop-type">(param: ColParams) => void</span> |   | Callback fired when a click event comes from a column header element. |
+| <span class="prop-name">onSortModelChange</span> | <span class="prop-type">(param: SortModelParams) => void</span> |   | Callback fired when the sort model changes before a column is sorted. |
+| <span class="prop-name">onPageChange</span> | <span class="prop-type">(param: PageChangeParams) => void</span> |   | Callback fired when the current page has changed. |
+| <span class="prop-name">onPageSizeChange</span> | <span class="prop-type">(param: PageChangeParams) => void</span> |   | Callback fired when the page size has changed. |
+| <span class="prop-name">onError</span> | <span class="prop-type">(args: any) => void</span> |   | Callback fired when an exception is thrown in the grid, or when the `showError` API method is called. |
+| <span class="prop-name">icons</span> | <span class="prop-type">IconsOptions</span> |   | Set of icons used in the grid. |
+
+The `ref` is forwarded to the root element.
+
+## CSS
+
+| Rule name | Global class | Description |
+|:-----|:-------------|:------------|
+| <span class="prop-name">root</span> | <span class="prop-name">.MuiDataGrid-root</span> | Styles applied to the root element.
+| <span class="prop-name">root</span> | <span class="prop-name">.MuiDataGrid-mainGridContainer</span> | Styles applied to the main container element.
```

Comment:
`root` is not implemented here and for all the other keys.

```suggestion
| <span class="prop-name"></span> | <span class="prop-name">.MuiDataGrid-mainGridContainer</span> | Styles applied to the main container element.
```

Actually, we could implement them with the context.